### DISCUSSION TITLE
openclash和V2RAYN测速为何相差那么多？高的是V2RAYN的 低的是opclash  以下是我的设置  请大佬看看

### DISCUSSION THREAD

**Discussion by w504925264** [vernesong/OpenClash#3478] [Q&A]
![屏幕截图 2023-08-26 135404](https://github.com/vernesong/OpenClash/assets/32632848/bfc96420-46ab-499a-afc8-9142e006a356)
![屏幕截图 2023-08-26 135739](https://github.com/vernesong/OpenClash/assets/32632848/6d2f884b-bdca-44a7-a7c4-2c1079fb11c6)
![屏幕截图 2023-08-26 135803](https://github.com/vernesong/OpenClash/assets/32632848/ee7edd
![屏幕截图 2023-08-26 135836](https://github.com/vernesong/OpenClash/assets/32632848/bf2db6b0-7
![屏幕截图 2023-08-26 135854](https://github.com/vernesong/OpenClash/assets/32632848/459797ab-f063-4d68-8dcc-a34acc95bc73)
da5-443b-9dac-da3eb5fdf171)
d5-5938-462b-94e0-13010d14c22f)
![屏幕截图 2023-08-26 135906](https://github.com/vernesong/OpenClash/assets/32632848/fbbdfaef-83a0-42f2-a646-acd886f1b6e1)
![屏幕截图 2023-08-26 135922](https://github.com/vernesong/OpenClash/assets/32632848/5b36311f-c506-4b07-8e1c-8130a5474c22)
![屏幕截图 2023-08-26 135945](https://github.com/vernesong/OpenClash/assets/32632848/9e467446-2615-4c0d-bb46-949de04e6fa2)

**Comment by maincooss** [comment]
搞定了吗，同问。

**Comment by refined-fish** [comment]
看看你的CPU占用满了没有，clash的负载是更大的，因为功能更细致，资源开销更大
